cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -274,17 +259,11 @@ message AvailablePackageSummary {
   // The name of the available package
   string name = 2;
 
-  // Latest available package version
+  // Latest available version
   //
   // The latest version available for this package. Often expected when viewing
   // a summary of many available packages.
-  string latest_pkg_version = 3;
-
-  // Latest available package app version
-  //
-  // The app version of the latest version available for this package. Often expected
-  // when viewing a summary of many available packages.
-  string latest_app_version = 4;
+  PackageAppVersion latest_version = 3;
```

Comment:
Don't know what's the proper way to go here. As the API is under heavy development, shouldn't we just shift the sequence to 3, 4, 5... instead of skipping the field `4` ?
I know that's for backwards-compatible changes, but here we are far from something stable yet, IMHO :P

Reply:
Yeah, I've done that now (in the previous PR).